app/models/topic.rb

```diff
@@ -1924,7 +1924,9 @@ def is_category_topic?
 
   def reset_bumped_at(post_id = nil)
     post =
-      (
+      if post_id.is_a?(Post)
```

Comment:
```suggestion
      if post_or_post_id.is_a?(Post)
```

We usually name the argument something like the above to make it clear that the argument can both be an instance of `Post` or `Post#id`.